Mathlib/Data/Nat/Defs.lean

```diff
@@ -137,7 +137,12 @@ lemma one_lt_iff_ne_zero_and_ne_one : ∀ {n : ℕ}, 1 < n ↔ n ≠ 0 ∧ n ≠
 
 lemma le_one_iff_eq_zero_or_eq_one : ∀ {n : ℕ}, n ≤ 1 ↔ n = 0 ∨ n = 1 := by simp [le_succ_iff]
 
-@[simp] lemma lt_one_iff : n < 1 ↔ n = 0 := Nat.lt_succ_iff.trans <| by rw [le_zero_eq]
+#adaptation_note
+/-- This was accidently left off when upstreamed in Lean in
+https://github.com/leanprover/lean4/pull/5478.
+It can be removed again after nightly-2024-09-28.
+-/
+attribute [simp] lt_one_iff
```

Comment:
Given that this is adapations for nightly 2024-09-30, can't we remove this now?

Reply:
Good catch. (There was a delay doing this adaptation because of an unfixable failure until this nightly arrived.)